src/lib/django-activities/activities/notifiers/oauth/base.py

```diff
@@ -74,7 +74,7 @@ def get_credentials(cls, filename):
         The return value is a dictionary to initialize a
         requests_oauthlib.OAuth1
         """
-        if not os.path.exists(filename):
+        if not os.path.exists(filename) and not settings.TESTING:
```

Comment:
In that case, you should not control the flow with this way.
Instead, you can suppress certain warnings by [warnings API](https://docs.python.org/3/library/warnings.html#warning-filter). Prob, write such code in test's initialization code.